### PR TITLE
Add kinetic package for Google Dialogflow ROS wrapper

### PR DESCRIPTION
**Pull request by piraka9011**
Please add the dialogflow_ros package for documentation and indexing on ros.org.

### CODE REVIEW

File: kinetic/distribution.yaml [hunk 1]
```diff
@@ -1795,6 +1795,16 @@ repositories:
       url: https://github.com/ros/diagnostics.git
       version: indigo-devel
     status: maintained
+  dialogflow_ros:
+    doc:
+      type: git
+      url: https://github.com/piraka9011/dialogflow_ros.git
+      version: kinetic-devel
```
Comment (mikaelarguedas) [inline kinetic/distribution.yaml]: CI fails because there is no branch named `kinetic-devel` on that repository (same below)

Comment (piraka9011) [reply kinetic/distribution.yaml]: This has been fixed, can you try again?

**Comment by mikaelarguedas** [comment]
re triggering travis...

**Comment by mikaelarguedas** [comment]
CI passed :+1:

**Pull request merged**